ballista/rust/core/proto/datafusion.proto

```diff
@@ -176,6 +176,7 @@ enum ScalarFunction {
   Translate=60;
   Trim=61;
   Upper=62;
+  UserDefined = 63;
```

Comment:
Does this cover all user-defined code or do we need to specialize for user-defined function vs user-defined aggregate functions?

Reply:
This PR in general only handles Scalar UDFs, but it's a good thing you asked because this was actually a relic of a different encoding I was trying first and was left in by accident :). Removed it.